docs/architecture-decisions/multiple-sync-sources.md

```diff
@@ -0,0 +1,82 @@
+---
+# Valid statuses: draft | proposed | rejected | accepted | superseded
+status: draft
+author: Dave Josephsen
+created: 2025-05-21
+updated: 2025-05-21
+---
+
+# ADR: Multiple Sync Sources
```

Comment:
Sorry, I still don't get the high-level idea after reading the ADR - are we trying to change something for config syncing or flag (and other data) storage of flagd?